powershell/plugins/create-commands-v2.ts

```diff
@@ -170,11 +170,43 @@ export /* @internal */ class Inferrer {
 
     this.state.message({ Channel: Channel.Debug, Text: 'detecting high level commands...' });
     for (const operationGroup of values(model.operationGroups)) {
+      let hasPatch = false;
+      let getOperation: Operation | undefined, putOperation: Operation | undefined;
+      let pathLength = -1;
       for (const operation of values(operationGroup.operations)) {
+        if (operation.requests?.[0]?.protocol?.http?.method.toLowerCase() === 'patch') {
```

Comment:
I think we will need to refactor the for loop here.
1. If hasPatch is true, no get->put, otherwise go to 2
2. If putOperation does not exist, no get->put, otherwise go to 3
3. If getOperation and get path is equal to the put path, we should support get->put

Reply:
please see line 200